scripts/python/check_sv_updates.py

```diff
@@ -0,0 +1,160 @@
+"""
```

Comment:
Add python3 shebang:

```suggestion
#!/usr/bin/env python3                                                          

"""
```